src/Forms/Main.cs

```diff
@@ -1088,10 +1088,16 @@ private void Main_Load(object sender, EventArgs e)
                 if (screen.Bounds.Height < Height)
                     Height = screen.Bounds.Height;
 
-                if (screen.Bounds.X + screen.Bounds.Width - 200 < Left)
-                    Left = screen.Bounds.X + screen.Bounds.Width - Width;
-                if (screen.Bounds.Y + screen.Bounds.Height - 100 < Top)
-                    Top = screen.Bounds.Y + screen.Bounds.Height - Height;
+                // Fix main window coordinate (Multi-Monitor issue)
+                if (screen.Bounds.Right < Left)

```

Comment:
This is how you produce it.
1. Start Subtitle Edit
2. Move it to secondary screen.
3. Close the entire app (while in secondary screen).
4. Unconnect HDMI cable (or just WinKey + P and choose PC Screen Only)
5. Re open subtitle edit.

After reopening Subtitle Edit  you won't be able to see the Main window, it will still be stuck in secondary screen boundary.